src/resources/Notification/Subscription/SubscriptionInterfaces.ts

```diff
@@ -15,19 +15,19 @@ export interface SubscriptionModel extends EditSubscriptionModel {
     id: string;
     lastModified: number;
     organizationId: string;
-    status: SubscriptionStatusModel;
+    status?: SubscriptionStatusModel;
     userId: string;
 }
 
 export interface SubscriptionParameters {
-    emailRecipients: string[];
-    emailSubject: string;
-    fromDisplayName: string;
-    serviceUrl: string;
+    emailRecipients?: string[];
+    emailSubject?: string;
+    fromDisplayName?: string;
+    serviceUrl?: string;
 }
 
 export interface SubscriptionPatternModel {
-    content: {additionnalProperties: string};
+    content: {[key: string]: any};
```

Comment:
```suggestion
    content: Record<string, unkown>;
```

Reply:
Why do we consider `unknown` instead of `any` in this case?

Reply:
The question is more why would we use `any` when we can use `unknown` https://stackoverflow.com/questions/51439843/unknown-vs-any